packages/browser-pool/src/abstract-classes/browser-controller.ts

```diff
@@ -244,7 +244,6 @@ export abstract class BrowserController<
      * @private
      */
     // @ts-expect-error Give runtime error as well as compile time
-    // eslint-disable-next-line space-before-function-paren, @typescript-eslint/no-unused-vars
     protected abstract async _newPage(pageOptions?: NewPageOptions): Promise<NewPageResult> {
```

Comment:
Not an issue in this PR, but considering its all in typescript now, we can probably remove the default implementation for abstract methods in the future (to also cleanup these comments)